src/sounddeviceportaudio.h

```diff
@@ -96,7 +96,7 @@ class SoundDevicePortAudio : public SoundDevice {
     ControlObjectSlave* m_pMasterAudioLatencyOverload;
     int m_underflowUpdateCount;
     static volatile int m_underflowHappend;
-    int m_nsInAudioCb;
+    qint64 m_nsInAudioCb;
```

Comment:
OK.
Just remember: "int" is always 32 bit on our targets, "long" is the one that matters here.
Still open: https://bugs.launchpad.net/mixxx/+bug/1094143